core.md

```diff
@@ -0,0 +1,125 @@
+
+You may [apply to join some teams](https://links.neoforged.net/apply), but some require appointment from within an existing team, or further voting after application.
+
+#### Steering Council
+
+The Steering Council is the overall management and oversight of the Project. 
+
+They “steer” the project to keep it aligned with the long-term vision, hence its secondary name of Vision Team.
+
+The Council has at all times an odd number of active members, such that stalemates are not possible.
+
+A member may choose to become inactive and appoint another in their place until they return.
+
+At least one member of the Council is promoted from other teams via an annual internal vote, to ensure that the Council always has fresh insight from a new team member with a different view on situations, and thus does not stagnate over time.
+
+Once a year, [an internal voting process](https://peps.python.org/pep-8104/) will determine the members that will be promoted to the Steering Council, via the ranked voting system. It is intended that at least one, and at most three members will be promoted at a time.
+
+The Council also serves double duty as a mediator for disagreements between other teams. These shall be resolved by discussion, rather than by unanimous applications of the rules, so this is not in the purview of this document.
+
+However, the Council does not have all-encompassing powers. They are the avenue of last resort, and the vast majority of decisions should be made by the team in charge, not by the Steering Council.
+
+The Council has these powers to resolve the 1% of exceptional situations, not the 99%.
+
+#### Maintainers
+
+The Maintainers team consists of all project developers who have push access or technical rights to the project's repositories or infrastructure.
+
+They are the ones keeping the Project's repositories up to date, and ensuring the quality and standards of the Project are upheld, as well as adding new features as the community comes to require them.
+
+Though Maintainers may individually or collectively have different access rights across the project's many repositories or infrastructure services, all Maintainers within the team are considered equal to each other with respect to their rights within the team's governance.
+
+Maintainers may be appointed by other Maintainers, or by application from the person who wishes to join the team.
+
+However, they have no jurisdiction or weight in moderating or managing the community, unless the Maintainer in question is also a Moderator.
+
+
+#### Moderators
+
+The Moderation team is composed of all moderators of all community spaces (GitHub, Discord) under the purview of the project.
+
+They have the responsibility of enforcing the policies and rules of each space, as well as the Code of Conduct, and applying disciplinary sanctions to people who violate said rules.
+
```

Comment:
How are moderators appointed? Do they also have the same process as maintainers? Or are they slightly different?

Reply:
Good point! They share the same application system.